src/v2_services/chat_service.py

```
@@ -411,9 +434,74 @@ async def generate_answer(
+    if session_id:
+        save_to_history(session_id, search_query, full_answer)
+
+    # 메타데이터 전송
+    meta = json.dumps(
+        {
+            "related_law_id_list": law_ids,
+            "search_success": True,
+        }
+    )
+    yield f"data: [META]{meta}\n\n"
+    yield f"data: [DONE]\n\n"
```

Ruff (0.15.4)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)